CHANGELOG.md

```diff
@@ -68,6 +68,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased 2.x]
 ### Added
+- Added feature to check per awareness attribute value health([#5231](https://github.com/opensearch-project/OpenSearch/pull/5231))
```

Comment:
I think we're adding a new API here, so it's not a "feature to check", no?

Reply:
Make sense will make that change